src/main/java/com/turkcell/staj/entities/User.java

```diff
@@ -1,4 +1,28 @@
 package com.turkcell.staj.entities;
 
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "surname")
+    private String surname;
+    @Column(name = "balance")
+    private double balance;
+    @OneToMany(mappedBy = "user")
+    private List<Transaction> transactions;
 }
```

Comment:
reviews tablosu ile FK ilişkinin tanımlanabilmesi için aşağıdaki satır user entity'e eklenmelidir.
`@OneToMany(mappedBy = User)
List<Review> reviews;`